.changeset/chilled-papayas-admire.md

```diff
@@ -0,0 +1,63 @@
+---
+"@jspsych/config": minor
```

Comment:
did this package actually change? and what about the extension-* packages?

Reply:
Removed this, extensions-* and test-utils from changelog.